store/tikv/tests/rawkv_test.go

```diff
@@ -188,6 +187,8 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 	c.Assert(err, NotNil)
 }
 
+var rawBatchPutSize = tikv.ConfigProbe{}.GetRawBatchPutSize()
```

Comment:
I suggest replace https://github.com/pingcap/tidb/blob/b5fab43a93d3594d08acd3eb5d078bee2a7e40f1/store/tikv/tests/rawkv_test.go#L197 with `tikv.ConfigProbe{}.GetRawBatchPutSize()` and remove this definition.

Reply:
ok